Detectors/ITSMFT/ITS/simulation/src/V3Layer.cxx

```diff
@@ -3261,7 +3204,7 @@ TGeoVolume *V3Layer::createModuleOuterB(const TGeoManager *mgr)
 //
 
 
-  char volname[30];
+  char chipName[30], sensName[30], volName[30];
```

Comment:
`std::string` also here or at least constant variable for array size, used here and in `snprintf`